libr/debug/debug.c

```diff
@@ -1091,7 +1091,7 @@ R_API int r_debug_continue_kill(RDebug *dbg, int sig) {
 		return false;
 	}
 #if __WINDOWS__
-	r_cons_break_push (w32_break_process, dbg);
+	r_cons_break_push (w32_break_process_wrapper, dbg);
```

Comment:
if my ^F works fine this function is only used in debug.c, so make it static

Reply:
Yes you're right which is why I made it a wrapper function.